src/main/webapp/app/assessment/assessment-detail/assessment-detail.component.ts

```diff
@@ -42,4 +42,17 @@ export class AssessmentDetailComponent {
     public get sanitizedText(): string {
         return sanitize(this.text);
     }
+    allowDrop(ev: any) {
+        ev.preventDefault();
+    }
+    drop(ev: any) {
```

Comment:
You might want to refactor the duplicated code for `allowDrop()` and `drop()` **(1)**.

Reply:
This `drop()` is for the feedback elements with reference. The other methods are for general feedback. I refactored the general feedback drop method further and used the assessment-general-feedback component for the text and file-upload exercises as well. Thank you for noticing this.